SPEC.md

```diff
@@ -176,16 +210,16 @@ line of the file.
 
 Offsets, however are 0-based. A Position of `{ "offset": 4 }` represents the _fifth_ character in the file. Importantly, the `offset` is from the beginning of the file, not the beginning of a line. Newline characters (and all characters) count when computing an offset.
 
-### Contents
+#### Content
```

Comment:
What was the thought behind singularizing this but pluralizing other single-value fields like, say, `Descriptions`? Were you just avoiding the id-collision with the TOC up top? That seems like a tech limitation we should think about.

I'd personally vote for making the TOC header actually "Table of Contents" for now, though, so this can be consistent with the other issue-field-subheaders here. Or, alternatively, write the TOC header as actual HTML (`<h2 id="toc">Contents</h2>`) to avoid the collision.